src/3id/index.js

```diff
@@ -233,7 +239,7 @@ class ThreeId {
       return utils.callRpc(this._provider, '3id_encrypt', { message, space, to })
     } else {
       const keyring = this._keyringBySpace(space)
-      let paddedMsg = utils.pad(message)
+      let paddedMsg = typeof message === 'string' ? utils.pad(message) : message
```

Comment:
This doesn't seem correct. If we have an object we should do something like: `utils.pad(JSON.stringify(message))`. Not sure how an object would be handled here otherwise.

Reply:
that is for uint8 arrays/buffer, so assumes string or that here, (considered making separate func, but reused more code here instead) could make more specific than string or no string

Reply:
I see, the way you use it it to encrypt the buffer of the encKey. Makes sense. I guess the way I had implemented this before wasn't general enough to deal with things other than strings.

Reply:
yeah exactly, may split in future, but think makes sense to leave shared code path for now